visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.po.ts

```diff
@@ -10,7 +10,17 @@ export class NodeContextMenuPageObject {
 		await page.waitForSelector("#loading-gif-map", { visible: false })
 	}
 
+	async isOpened() {
+		const menu = await page.waitForSelector("#codemap-context-menu", { visible: true })
+		const isHidden = await menu.evaluate(element => { return element.getAttribute("aria-hidden") })
+
+		return isHidden !== 'true'
+	}
+
 	async isClosed() {
-		return page.waitForSelector("node-context-menu-component", { visible: false })
+		const menu = await page.waitForSelector("#codemap-context-menu", { hidden: true })
+		const isHidden = await menu.evaluate(element => { return element.getAttribute("aria-hidden") })
```

Comment:
Aria attributes are used for e.g., screen readers. Do we actually always use these at all?